src/modules/amp-caching/index.ts

```diff
@@ -1,3 +1,28 @@
+import _workbox from 'workbox-sw';
+declare const workbox: typeof _workbox;
```

Comment:
@kristoferbaxter this will go away in the following PR as the bundled workbox is working